pkg/windows/windows.go

```diff
@@ -550,6 +563,9 @@ func (vm *windows) changeHostName() error {
 
 // createDirectories creates directories required for configuring the Windows node on the VM
 func (vm *windows) createDirectories() error {
+	if vm.runtime == containerdRuntime {
+		RequiredDirectories = append(RequiredDirectories, containerdDir)
```

Comment:
This wont be tested here: https://github.com/openshift/windows-machine-config-operator/blob/c7090d62ed3af48c5114a49324639a3639b71781/test/e2e/delete_test.go#L85

Reply:
thanks for the catch, added the containerdDir as discussed.